other_procore_apis/bim_web_viewer.md

```diff
@@ -832,13 +791,541 @@ Manually dispatches an event
+| message     | string | A human readable message.                         |
+
+---
+
+### measured
+
+<p class="heading-link-container"><a class="heading-link" href="#measured"></a></p>
+
+Fires when measuring two objects has completed.
+
+---
+
+### modelTreeParentInteracted
+
+<p class="heading-link-container"><a class="heading-link" href="#modeltreeparentinteracted"></a></p>
+
+Fires when an object in the `Model Objects` has been interacted with.
+
+#### Data Properties
+
+| Field Name | Type   | Description                                                                                                                                                                   |
+| ---------- | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| children   | number | The number of children the interacted object has.                                                                                                                             |
+| action     | string | The type of action, can be the following: `checkbox`, `open`, `click`, `double_click` or `close`                                                                              |
+| depth      | number | The depth of the object interacted with. A depth of 0 indicates top level objects. A depth of 1 indicates an object that is one level down from root level objects and so on. |
+
+---
+
+### markupDisplayed
+
+<p class="heading-link-container"><a class="heading-link" href="#markupdisplayed"></a></p>
+
+Fires when markup from a viewpoint has been displayed.
+
+---
+
+### markupDismissed
+
+<p class="heading-link-container"><a class="heading-link" href="#markupdismissed"></a></p>
+
+Fires when markup from a viewpoint has been dismissed.
+
+---
+
+### navigationChanged
+
+<p class="heading-link-container"><a class="heading-link" href="#navigationchanged"></a></p>
+
+Fires when the navigation mode has changed.
+
+#### Data Properties
+
+| Field Name | Type   | Description                                     |
+| ---------- | ------ | ----------------------------------------------- |
+| mode       | number | The type of navigation that was just activated. |
+| controls   | object | An instance of the controller object.           |
+
+---
+
+### objectDoubleClick
+
+<p class="heading-link-container"><a class="heading-link" href="#objectdoubleclick"></a></p>
+
+Fires on double mouse clicks. If the double mouse click occurs on an object, this event returns a Meshnode ID, otherwise -1 is returned. A double click also produces one `objectSingleClick` event.
+
+---
+
+### objectSingleClick
+
+<p class="heading-link-container"><a class="heading-link" href="#objectsingleclick"></a></p>
+
+Fires on a single mouse click. If the single mouse click occurs on an object, this event returns Meshnode ID, otherwise -1 is returned.
```

Comment:
```suggestion
Fires on a single mouse click. If the single mouse click occurs on an object, this event returns a Meshnode Index, otherwise -1 is returned.
```